qdrant/v1.2.x/points.md

```diff
@@ -318,6 +318,59 @@ Currently, you would need to re-upload the point to modify the vector.
 
 The second is to modify the payload, for which there are several methods.
 
+### Update vectors
+
+*Available since v1.2.0*
```

Comment:
Available as of v1.2.0. Since is used in other cases.

Reply:
Currently the documentation only uses 'since' versus 'as of'.

Reply:
I agree, I'd prefer a PR changing all occurrences in one go :+1: